server/src/test/java/com/objectcomputing/checkins/services/memberprofile/currentuser/CurrentUserControllerTest.java

```diff
@@ -37,20 +42,15 @@ public class CurrentUserControllerTest {
     @Mock
     CurrentUserServices currentUserServices;
 
-    @Mock
-    RoleServices roleServices;
-
-    @Mock
-    PermissionServices permissionServices;
 
     @Inject
     CurrentUserController currentUserController;
 
+    @Inject
+    EmbeddedServer embeddedServer;
+
     @BeforeAll
     void setup() {
-        userAttributes.put("name", firstName + ' ' + lastName);
-        userAttributes.put("email", userEmail);
-        userAttributes.put("picture", imageUrl);
 
         MockitoAnnotations.initMocks(this);
```

Comment:
Since `initMocks` is deprecated, could we change this to `openMocks`?

Reply:
I'd rather make that change in a separate PR, since we use `initMocks` across many tests and this instance wasn't part of the PR changes.